keyboards/cozykeys/bloomer/readme.md

```diff
@@ -0,0 +1,28 @@
+# Bloomer
+
+![Bloomer](http://assets.cozykeys.xyz/images/keyboards/bloomer/bloomer-angle-2_1600x1600.jpg)
```

Comment:
```suggestion
![Bloomer](http://assets.cozykeys.xyz/images/keyboards/bloomer/bloomer-angle-2_800x800.jpg)
```

Reply:
Is there a reason not to link to the higher res image?

Reply:
You can link to it but the embed itself should not be unreasonably large.